source/globalCommands.py

```diff
@@ -4289,6 +4290,37 @@ def script_cycleParagraphStyle(self, gesture: "inputCore.InputGesture") -> None:
 		config.conf["documentNavigation"]["paragraphStyle"] = newFlag.name
 		ui.message(newFlag.displayString)
 
+	def cycleOutputAudioDevices(self, direction: int):

```

Comment:
Instead of using an int for direction, why not make it default to forward? Then you could use True/False, with a bool type defaulting to True.
Rename it from `direction: int` to `cycleForward: bool = True` or similar.